_studio/shared/include/mfx_config.h

```diff
@@ -85,6 +85,10 @@
     //#define MFX_ENABLE_HEVCE_DIRTY_RECT
 #endif
 
+#if defined(MFX_ENABLE_H265_VIDEO_DECODE)
+    #define MFX_ENABLE_HEVCE_HDR_SEI
```

Comment:
MFX_ENABLE_HEVC**E**_HDR_SEI ??

Reply:
Have been removed because there is no need for Marco protection in the pure SW feature.